packages/remix/src/build.ts

```diff
@@ -192,11 +192,9 @@ export const build: BuildV2 = async ({
     // If we are, prepend the app root directory from config onto the build path.
     // e.g. `/apps/my-remix-app/api/index.js`
     const isMonorepo = repoRootPath && repoRootPath !== workPath;
-    if (isMonorepo && config.projectSettings?.rootDirectory) {
-      serverBuildPath = join(
-        config.projectSettings.rootDirectory,
-        serverBuildPath
-      );
+    if (isMonorepo) {
+      const rootDirectory = relative(repoRootPath, workPath);
+      serverBuildPath = join(rootDirectory, serverBuildPath);
```

Comment:
I think it should be sufficient to say that `rootDirectory` is the same as `workPath` here

Reply:
Keeping it as-is because we want to pass a relative path, not absolute path.